### PR TITLE
[docs] Fix minor issues such as typos and grammatical errors in multiple docs

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

While going through our documentation, I've found some issues (typos, formatting nits, and grammatical errors) on multiple pages.

Also, found out that on our home page, the number of Discount community members we reference is outdated.

# How

- Fix issues in multiple docs
- Update community members count for "Chat with the community" under Discover more section on home page.

# Test Plan

By going through issues and fixing them manually in each doc. To see the changes, use the diff to compare.

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/build-reference/easignore.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Ignoring files in .easignore
+title: Ignore files in .easignore
```
Comment (Simek) [inline docs/pages/build-reference/easignore.mdx:2]:
```suggestion
title: Ignore files via .easignore
```

Maybe? Feel free to skip.

File: docs/pages/eas-update/rollbacks.mdx [hunk 2]
```diff
@@ -5,7 +5,7 @@ description: Rollback a branch to a previous update or the embedded update.
 
 import { Terminal } from '~/ui/components/Snippet';
 
-> **info** Available for SDK 50 and above ([`expo-updates`](/versions/latest/sdk/updates/) >= 0.23.0).
+> **info** Available from SDK 50 and [`expo-updates`](/versions/latest/sdk/updates/) 0.23.0 and above.
```
Comment (Simek) [inline docs/pages/eas-update/rollbacks.mdx:8]:
```suggestion
> **info** Available from SDK 50, and with [`expo-updates`](/versions/latest/sdk/updates/) 0.23.0 and above.
```

```suggestion
> **info** Available from SDK 50 using [`expo-updates`](/versions/latest/sdk/updates/) 0.23.0 or above.
```

Maybe? Feel free to skip.

File: docs/ui/components/Home/sections/DiscoverMore.tsx [hunk 3]
```diff
@@ -60,7 +60,7 @@ export function DiscoverMore() {
           </div>
           <RawH3 className="!text-palette-blue11 !font-bold">Chat with the community</RawH3>
           <P className="!text-palette-blue11 !text-xs max-w-[32ch]">
-            Join over 20,000 other developers
+            Join over 40,000 other developers
```
Comment (Simek) [inline docs/ui/components/Home/sections/DiscoverMore.tsx:63]: 🔥

**Review by Simek — APPROVED** [review]
Thanks for the corrections, LGTM! 👍

File: docs/pages/build-reference/easignore.mdx (hunk 1 above)
Comment (amandeepmittal) [reply docs/pages/build-reference/easignore.mdx:2]: Good suggestion, thanks!

File: docs/pages/eas-update/rollouts.mdx [hunk 4]
```diff
@@ -5,7 +5,7 @@ description: Incrementally deploy updates on a new branch to a channel.
 
 import { Terminal } from '~/ui/components/Snippet';
 
-> **info** Available for SDK 50 and above ([`expo-updates`](/versions/latest/sdk/updates/) >= 0.23.0).
+> **info** Available from SDK 50 and [`expo-updates`](/versions/latest/sdk/updates/) 0.23.0 and above.
```
Comment (amandeepmittal) [inline docs/pages/eas-update/rollouts.mdx:8]:
```suggestion
> **info** Available from SDK 50 using [`expo-updates`](/versions/latest/sdk/updates/) 0.23.0 or above.
```

**Pull request merged**